presto-main/src/main/java/com/facebook/presto/split/PageSinkManager.java

```diff
@@ -46,19 +47,19 @@ public void removeConnectorPageSinkProvider(ConnectorId connectorId)
     }
 
     @Override
-    public ConnectorPageSink createPageSink(Session session, OutputTableHandle tableHandle)
+    public ConnectorPageSink createPageSink(Session session, OutputTableHandle tableHandle, PageSinkProperties property)
```

Comment:
`properties`

Reply:
I'll name them `pageSinkProperties` to be consistent with others.